packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -22,6 +22,28 @@ spec:
                   name: fluent-bit
         tls:
           enabled: {{ .Values.nodeAgents.tls.enabled | default false }}
+    {{- if .Values.additionalLoggingSources.rke.enabled }}
+    - name: win-agent-rke
+      profile: windows
+      nodeAgentFluentbit:
+        filterKubernetes:
+          Kube_Tag_Prefix: "kuberentes.C.var.lib.rancher.rke.log."
+        inputTail:
+          Path: "C:\\var\\lib\\rancher\\rke\\log"
+        extraVolumeMounts:
+        - source: "/var/lib/rancher/rke/log"
+          destination: "/var/lib/rancher/rke/log"
+          readOnly: true
+        daemonSet:
+          spec:
+            template:
+              spec:
+                containers:
+                - image: "{{ .Values.images.nodeagent_fluentbit.repository }}:{{ .Values.images.nodeagent_fluentbit.tag }}"
+                  name: fluent-bit
```

Comment:
Won't both of these daemonsets have same name, do they not clash ?

Reply:
This is fine because this is the container name _within_ the daemonset. Each nodeAgent is a daemonset